src/Slider.tsx

```diff
@@ -30,6 +31,7 @@ export const Slider: FC<Props> = ({
   label,
   value,
   onValueChange: onValueChangeProp,
+  onValueCommit: onValueCommitProp,
```

Comment:
OOI where are `onValueChangeProp` & `onValueCommit` prop defined?

Reply:
Mm, you're looking at a function parameter being destructured here. So they're defined right on these lines. It works like this:

```typescript
const foo = ({ value: inputValue }) => { // Function accepting an object with some entry named 'value'
  console.log(inputValue) // 'value' has been bound to the variable 'inputValue'
}

foo({ value: 999 }) // Logs '999'
```